components/customNodes.tsx

```diff
@@ -0,0 +1,168 @@
+import { Handle, Position } from "reactflow";
+import { ParticipantsMenu } from "./rules/participantsMenu.tsx";
+import IconGripVertical from "https://deno.land/x/tabler_icons_tsx@0.0.3/tsx/grip-vertical.tsx";
+import "flowbite";
+import "twind";
+
+// const handleStyle = { left: 2 };
+
+const getHandlePosition = (input: any) => {
```

Comment:
A proper type on input would be nice...e.g. `string` or even more specific: `"top" | "bottom" would be nice.

Also, a ternary versus if else would be nicer!